tests/src/JIT/Regression/JitBlue/GitHub_16041/GitHub_16041.il

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern mscorlib { auto }
+.assembly GitHub_16041 { }
+
+// The test originally hit a problem with "Extra flags on tree" on amd64,
+// because "fgMorphBlockOperand" did not set flags when morphing comma for addr exposed lcl_Var.
+// "StructY" is struct with exposed addr, it needs to be larger than 8 bytes for amd64.
+
+
+.class private auto ansi beforefieldinit Test
+       extends [mscorlib]System.Object
+{
+  .method private hidebysig static int32  Main() cil managed
+  {
+    .entrypoint
+    // Code size       10 (0xa)
+    .maxstack  8
+    .locals init (valuetype StructX V_0)
+    IL_0000:  ldloc      V_0
+    IL_0004:  call       void Test::Test(valuetype StructX)
+    IL_0009:  ldc.i4 100
+    IL_000a:  ret
+  } // end of method Test::Main
+
+  .method private hidebysig static void  Test(valuetype StructX StructX) cil managed noinlining
+  {
+    // Code size       10 (0xa)
+    .maxstack  8
+    .locals (valuetype StructY V_0)
+    IL_0000:  ldarga.s   StructX
+    IL_0002:  call       valuetype StructY Test::Convert(valuetype StructX*)
+    IL_0007:  stloc.s    V_0
+    IL_0009:  ret
+  } // end of method Test::Test
+
+  .method public hidebysig static valuetype StructY 
+          Convert(valuetype StructX* StructX) cil managed
+  {
+    // Code size       13 (0xd)
+    .maxstack  3
+    .locals (valuetype StructY V_0)
+    IL_0000:  ldloca.s   V_0
+    IL_0002:  ldarg.0
+    IL_0003:  conv.i
+    IL_0004:  ldc.i4.s   28
```

Comment:
@sandreenko is this test valid? I just saw it fail on CoreRT in the CI (looks nondeterministic, because it never failed before). This is doing a `cpblk` of 28 bytes from `StructX`, but `StructX` is only 12 bytes big and I don't see any guarantees about the destination of `cpblk` not being immediately after `StructX` - this would make them overlap and hit unspecified behavior in cpblk (per what ECMA-335 specifies about `cpblk`).

Reply:
This test came from this issue #16041, I have not checked `cpblk` instruction there. 
I think we can change struct `StructY` size, It should not affect the test behavior.

Reply:
We have not seen any failures on this test.
How often does it fail with corert?

Reply:
I've seen a single failure today, when running unoptimized code. Looking at the project file for this test, seems like CoreCLR always runs this optimized?

Reply:
https://ci.dot.net/job/dotnet_corert/job/master/job/debug_windows_nt/17/